cypress_tests/cypress/integration/polls/polls_with_logged_in_users.spec.js

```diff
@@ -0,0 +1,18 @@
+describe("Polls for logged in users tests", () => {
+    const url = "/en/sections/questionnaire-testing/poll-only-for-logged-in-users/";
+
+    before("Login the user", () => {
+        cy.login("saqlain", "saqlain");
```

Comment:
Could you add another test where no user is logged in, to check that the "Login to participate" button is shown? Note that this was only merged in today, so you might have to merge/rebase onto develop.